Nef_3/include/CGAL/Nef_3/polygon_mesh_to_nef_3.h

```diff
@@ -126,48 +126,44 @@ class Face_graph_index_adder<CGAL::SNC_indexed_items, PolygonMesh, SNC_structure
 
   typedef Halfedge_around_face_circulator<PolygonMesh>
     Halfedge_around_facet_const_circulator;
-    typedef std::vector<SHalfedge_handle> Hash;
+    typedef std::vector<SHalfedge_handle> SHalfedge_list;
```

Comment:
It's only a local variable, but I am not sure that it is a good idea to call a vector list.

Reply:
I can update this. Should it be:
* `SHalfedges`
* `SHalfedge_vector`
* `SHalfedgeVector`

Reply:
I would go for `SHalfedges`